### PR TITLE
[FLINK-35577] Fix the CI environment for FRocksDB-8.10

### PR DESCRIPTION
**Pull request by mayuehappy**
Setup base branch for FrocksDB-8.10.0
cherry-pick from https://github.com/ververica/frocksdb/pull/70

### CODE REVIEW

**Review by rkhachatryan — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -641,26 +658,26 @@ jobs:
...
+#          name: "Set Java Environment"
+#          command: |
+#            echo "JAVA_HOME=${JAVA_HOME}"
+#            echo 'export PATH=$JAVA_HOME/bin:$PATH' >> $BASH_ENV
+#            which java && java -version
+#            which javac && javac -version
+#      - run:
+#          name: "PMD RocksDBJava"
+#          command: make V=1 J=8 -j8 jpmd
+#      - post-pmd-steps
```
Comment (Zakelly) [inline .circleci/config.yml:680]: So we don't need this stage any more, right?

**Review by Zakelly — COMMENTED** [review]
Thank for the PR!

File: .circleci/config.yml (hunk 1 above)
Comment (mayuehappy) [reply .circleci/config.yml:680]: Yes, unless we add the maven-pmd-plugin related parts in `pom.xml.template`.
we can add it in the following commit if necessary. 
Right now , i think it's no needed.

**Review by Zakelly — APPROVED** [review]
LGTM merging...

**Pull request merged**